### ISSUE TITLE
Download tabula for Mac (OSX El Capitan) but application will not open 

### ISSUE THREAD

**Issue by zanzibarUTS** [tabulapdf/tabula#439]

**Comment by jazzido** [comment]
Hi @zanzibarUTS,

We'd need to take a look at the error messages. Please open a Terminal, an execute Tabula from there. If you downloaded and unzipped Tabula to your Downloads folder, run this command:

```
~/Downloads/tabula/Tabula.app/Contents/MacOS/JavaAppLauncher
```

Please include here the output produced by the program.

**Comment by zanzibarUTS** [comment]
Hello
I tried two different ways.
First downloading Tabula and putting it in my applications folder. I click
on the app (I double checked it is not an alias) and nothing happens, the
app icon does not pop up in my dock either)

Second, I ran it as you said via terminal from downloads.

Images attached for your reference including the error message.

Pardon me if this is an obvious problem which I have overlooked, as I am
certainly no coder.

Regards and thanks for your help,
Henry


_______________________
Henry Zwartz

http://www.henryzwartz.com/
+61 418 293 173
linkedin.com/henryzwartz
<https://www.linkedin.com/pub/henry-zwartz/30/125/568>
_____________________

The information contained in this email message and any accompanying files
is confidential. If you are not the intended recipient, any use,
dissemination, reliance, forwarding, printing or copying of this email or
any attached files is unauthorised. No part of it should be reproduced,
adapted or communicated without the written consent of the copyright owner.

**Comment by jazzido** [comment]
@zanzibarUTS, unfortunately github drops attachments when replying to an issue via email. Please post your image again through the web interface.

Thanks!

**Comment by zanzibarUTS** [comment]
Hello
I tried two different ways.
First downloading Tabula and putting it in my applications folder. I click on the app (I double checked it is not an alias) and nothing happens, the app icon does not pop up in my dock either)

Second, I ran it as you said via terminal from downloads.

Images attached for your reference including the error message.

Pardon me if this is an obvious problem which I have overlooked, as I am certainly no coder.

Regards and thanks for your help,
Henry

<img width="572" alt="screen shot 2015-12-06 at 10 46 38 am" src="https://cloud.githubusercontent.com/assets/16144236/11610942/65384b72-9c08-11e5-96d5-ae4fbfb8b330.png">
<img width="913" alt="screen shot 2015-12-06 at 10 46 06 am" src="https://cloud.githubusercontent.com/assets/16144236/11610944/65398442-9c08-11e5-8404-13b26066592f.png">
<img width="578" alt="screen shot 2015-12-06 at 10 44 45 am" src="https://cloud.githubusercontent.com/assets/16144236/11610945/653aa32c-9c08-11e5-840f-6f9f074a1ccb.png">
<img width="907" alt="screen shot 2015-12-06 at 10 43 54 am" src="https://cloud.githubusercontent.com/assets/16144236/11610943/65393e92-9c08-11e5-9a2d-c175d8ce6847.png">
